content/docs/sync/synchronisation.md

```diff
@@ -16,23 +16,25 @@ top = false
 
 ## Viewing the synchronisation status
 
-To view the synchronisation status, go to `Sync` in the lower section of the navigation panel: 
+To view the synchronisation status, go to `Sync` in the lower section of the navigation panel:
 
 ![sync: nav](/docs/sync/images/sync_nav.png)
 
-The page shows the number of records waiting to be pushed to the central server and the time that the last sync happened.
+The page shows the number of records waiting to be pushed to the central servers, and the time that the last sync happened.
 
 ![sync: status](/docs/sync/images/sync_status.png)
 
+As of v2.0.0, your Open mSupply server actually syncs to two central servers! The `Push V6` and `Pull V6` steps transfer data to and from the Open mSupply Central Server, which you can read more about [here](../../introduction/central). The `Push`, `Pull Central` and `Pull Remote` steps are syncing with an mSupply central server.
```

Comment:
Caps in one central server and not in the other... honestly not sure how to handle this since on one hand I do want it caps but on the other it's not caps anywhere else...

Reply:
...change to lower case then?

Reply:
Yes please (: